NativeScript/NativeScript.mm

```diff
@@ -59,25 +109,20 @@ - (instancetype)initWithConfig:(Config*)config {
         }
     }
     
-    return self;
-    
 }
 
-- (void)runMainApplication {
-    runtime_->RunMainScript();
-
-    CFRunLoopRunInMode(kCFRunLoopDefaultMode, 0, true);
+- (instancetype)initWithConfig:(Config*)config {
+    if (self = [super init]) {
+        [self initializeWithConfig:config];
+    }
```

Comment:
init will be called twice if this method is called (one here and another in initializeWithConfig)